service/history/configs/task.go

```diff
@@ -30,11 +30,18 @@ import (
 	"go.temporal.io/server/common/tasks"
 )
 
-var DefaultTaskPriorityWeight = map[tasks.Priority]int{
-	tasks.PriorityHigh:   10,
-	tasks.PriorityMedium: 2,
-	tasks.PriorityLow:    1,
-}
+var (
+	DefaultActiveTaskPriorityWeight = map[tasks.Priority]int{
+		tasks.PriorityHigh: 10,
```

Comment:
Not related to this PR. It seems like we treat 0 as the highest priority in other places. Any reason we do a reverse order here?

Reply:
The number here is the weight of the priority, not priority itself. The number for priority is still the smaller the number the higher the priority.